src/System.ComponentModel.Composition/tests/System/ComponentModel/IO/TemporaryDirectory.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+namespace System.IO 
+{
+    public class TemporaryDirectory : IDisposable
```

Comment:
Delete this file and replace with using src\Common\tests\System\IO\TempDirectory.cs

Reply:
But you may not need it a all (nor TempFile) since if you derive from FileCleanupTestBAse it does the cleanup for you.